tests/test_filter_pydantic_sub_model.py

```diff
@@ -8,10 +8,20 @@
 class ModelB(BaseModel):
     username: str
 
+    @validator("username")
+    def check_username(cls, username, values):
```

Comment:
does adding this test new scenario? If the tests here don't trigger a failure case, I don't think adding these two validators adds anything, whereas testing for failure would make sure that the validators are inherited (worth checking in other places to make sure that isn't covered elsewhere, as this module seems to have a specific scenario in mind)